lexicons/app/bsky/feed/detach.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.detach",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining post URIs detached from a root post. The record key (rkey) of the detach record must match the record key of the root post in question, and that record must be in the same repository.",
+      "record": {
+        "type": "object",
+        "required": ["post", "targets", "updatedAt"],
+        "properties": {
+          "post": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to the post record."
+          },
+          "targets": {
+            "type": "array",
+            "maxLength": 50,
+            "items": {
+              "type": "string",
+              "format": "at-uri"
+            },
+            "description": "List of detached post URIs."
+          },
+          "updatedAt": { "type": "string", "format": "datetime" }
```

Comment:
I assume this should get updated every time posts are added or removed from the `targets` array? should clarify in description.